Source/Core/Runtime/Serialization/NewtonsoftJsonSerializer/NewtonsoftJsonProcessSerializerV4.cs

```diff
@@ -13,6 +13,7 @@
 using System.Linq;
 using VRBuilder.Core.EntityOwners;
 using VRBuilder.Core.Behaviors;
+using UnityEngine;
```

Comment:
Should e change the Dokumentation of NewtonsoftJsonProcessSerializerV4 to include information about the new serialization strategy?

Reply:
Yes, done